src/components/List/List.tsx

```diff
@@ -149,7 +150,7 @@ export const List = chakra(
           __css={styles}
           {...rest}
         >
-          {title && <Heading id={`${id}-heading`}>{title}</Heading>}
+          {finalTitle}
```

Comment:
There are styles wrtten into the theme file specific for `h2`.  (For example, the 3px top border.)  Those styles are lost when something other than `h2` is used for the heading.  I am not sure what would be the best way to get those styles to apply to all heading levels.

Reply:
I see. I'll make this a broken style object and pass it to the heading function.